### DISCUSSION TITLE
primefaces-13.0.14-LTS-jakarta

### DISCUSSION THREAD

**Discussion by pdj-code** [primefaces/community#3110] [PrimeFaces] [answered]
I am new to charts. I am using PieChartModel, PieChartOptions and want to make legend label square instead of rectangle and make it align vertical at bottom of the chart and also change cursor to pointer cursor. how should I do ?

**Comment by DazzlingBob** [comment, marked as answer]
Something like (did not test this):

			// options - legend
			Legend legend = new Legend();
			legend.setDisplay(true);
			legend.setPosition("bottom");
			legend.setAlign("start");
			LegendLabel legendLabel = new LegendLabel();
			legendLabel.setBoxWidth(12);
			legend.setLabels(legendLabel);
			options.setLegend(legend);

I do however not recommend to use PieChartModel anymore.
Because it is deprecated in version 14.0.0 and marked for removal.
Instead use <p:chart> and raw JSON or XDEV Chart.js Java Model.

**Reply by pdj-code** [reply]
@DazzlingBob thanks for the help. I will try moving to <p:chart>.
